webapps/plf-meeds-extension/src/main/webapp/WEB-INF/conf/meeds/portal/meeds/pages.xml

```diff
@@ -53,29 +53,15 @@
         </container>
         <container id="PerkstoreWalletContainer" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
           <access-permissions>*:/platform/users</access-permissions>
-          <container id="PerkstoreContainer" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
+          <container id="PerkstoreContainer" template="system:/groovy/portal/webui/container/UIAddOnColumnContainer.gtmpl">
```

Comment:
please use more specific name like, `PerkstoreWidgetContainer`, same for WalletContainer

Reply:
Done!